src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -25,12 +28,24 @@ class TimezoneType extends AbstractType implements ChoiceLoaderInterface
      * The choices are generated from the ICU function \DateTimeZone::listIdentifiers().
      *
      * @var ArrayChoiceList
+     *
+     * @deprecated since version 3.4, to be removed in 4.0
      */
     private $choiceList;
 
     /**
      * {@inheritdoc}
      */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        if ('datetimezone' === $options['input']) {
```

Comment:
i meant `if ('datetimezone' === strtolower($options['input'])) {`..

then again im not sure using class-ish values for `input` makes sense (compared to `data_class`). It kinda implies it supports class names, which it doesnt.